README_JEDI.md

```diff
@@ -1,28 +1,28 @@
 README_JEDI.md
 
-CRTM REL-3.0.0 
-Released April 1, 2021
+CRTM REL-3.0.1  Released September 23, 2023
+CRTM REL-3.0.0  Released April 1, 2021
 
 
 
 The README.md file contains a lot of general information about this repository and the legacy build system based on autotools.
 
-CRTM REL-3.0.0 JEDI environment build instructions
+CRTM REL-3.0.1 JEDI environment build instructions
 =========================================================
 
 Preamble
 --------
 
-CRTM v3.0.0-alpha release (`REL-3.0.0`)  
+CRTM v3.0.1 release (`REL-3.0.1`)  
 
-This is a fully functional release of CRTM v3.0.0-alpha. 
+This is a fully functional release of CRTM v3.0.1.
 
 Basic requirements:  
 (1) A Fortran 2003 compatible compiler.  
 (2) A netCDF4 / HDF5 library.   
 (3) A linux, macOS, or unix-style environment.  This has not been tested under any Windows Fortran environments.
 (4) Bash shell is preferred.
-(5) A suitable JCSDA JEDI environment: either HPC enabled, a JEDI container, JEDI-stacks, or at a bare minimum ecbuild from ectools (google it). 
+(5) A suitable JCSDA JEDI environment: either HPC enabled, a JEDI container, JEDI-stacks, or at a bare minimum ecbuild from ectools OR cmake.
```

Comment:
```suggestion
(5) A suitable JCSDA JEDI environment: either HPC enabled, a JEDI container, spack-stack, or at a bare minimum ecbuild from ectools OR cmake.
```
Should ecbuild be removed here?

Reply:
ecbuild will still build a cmake-only CMakeLists.txt file, so if they have / use ecbuild as part of their normal workflow, the present change should not impact them.